getting-started/best-practices.mdx

```diff
@@ -49,17 +49,23 @@ While Rive provides a Lottie converter for easy .riv file export, recreating gra
 
 Working directly in Rive is generally preferable as it allows for file optimization based on your animation requirements. For instance, using bones and constraints to create rigs will result in fewer keys in the animation compared to converting directly from Lottie.
 
+### Layer Blend Modes for Web
```

Comment:
This is mostly true for WebGL2 (Rive Renderer) I believe!

I've been meaning to add a note about this in the web section of our docs too. Basically if you choose to use the Rive Renderer to speed up other areas, you need to account for this blend mode perf issue.

This is something that is not a problem if you are using the experimental chrome flags to enable pixel local storage. It's only a problem with the MSAA fallback and not something we can fix afik.